frontend/model/contracts/chatroom.js

```diff
@@ -265,9 +263,7 @@ sbp('chelonia/defineContract', {
       }
     },
     'gi.contracts/chatroom/leave': {
-      validate: objectOf({
-        memberID: optional(string) // member to be removed
-      }),
```

Comment:
In the future, I ask that you please avoid making these sorts of modifications to the code, as it adds more work for me for no reason and also increases the likelihood of merge conflicts

Reply:
I will revert this change too since I am going to make a commit to resolve your previous feedback.